### PR TITLE
Add doc for model deviation

### PR DESCRIPTION
**Pull request by AnguseZhang**

### CODE REVIEW

File: doc/third-party/lammps-command.md [hunk 1]
```diff
@@ -18,7 +18,10 @@ The DeePMD-kit package provides the pair_style `deepmd`
 pair_style deepmd models ... keyword value ...
 ```
 - deepmd = style of this pair_style
-- models = frozen model(s) to compute the interaction. If multiple models are provided, then the model deviation will be computed
+- models = frozen model(s) to compute the interaction. 
+If multiple models are provided, then only the first model serves to provide energy and force prediction for molecular dynamics, 
+and the model deviation will be computed among all models.
```
Comment (njzjz) [inline doc/third-party/lammps-command.md:23]: The model deviation will be only computed every `out_freq` steps.

**Review by denghuilu — APPROVED** [review]

File: doc/third-party/lammps-command.md (hunk 1 above)
Comment (denghuilu) [reply doc/third-party/lammps-command.md:23]: Agree

Comment (amcadmus) [reply doc/third-party/lammps-command.md:23]: @njzjz plz suggest a change.

File: doc/third-party/lammps-command.md [hunk 2]
```diff
@@ -18,7 +18,10 @@ The DeePMD-kit package provides the pair_style `deepmd`
...
+
```
Comment (njzjz) [inline doc/third-party/lammps-command.md:24]:
```suggestion
If multiple models are provided, then only the first model serves to provide energy and force prediction for each timestep of molecular dynamics, 
and the model deviation will be computed among all models every `out_freq` timesteps.
```

**Review by amcadmus — APPROVED** [review]

**Pull request merged**